style.css

```diff
@@ -2,3 +2,203 @@
   padding: 0;
   margin: 0;
 }
+body{
+  color: #797979;
+  font-family: 'Source Sans Pro', sans-serif;
+}
+
+footer {
+  width: 100%;
+  height: 600px;
+  background-color: #000000;
+}
+footer h6{
+  color: #ffffff;
+  font-size: 18px;
+  margin-top: 70px;
+  
+}
+
+footer p{
+  color: #797979;
+  font-size: 17px;
+  
+}
+
+.footer-input-btn{
+  background-color: #222;
+  color: #ff9907;
+  height: 30px;
+  width: 15%;
+  margin-left: 0;
+}
+
+.main-footer{
+  display: flex;
+  justify-content: space-between;   
+}
+.footer-content{
+  width: 25%;
+  margin: 30px; 
+}
+.footer-copyright{
+  margin-left: 400px;
+}
+
+
+.fimg{
+  width: 50px;
+  height: 50px;
+  padding: 2px;
+}
+.firstc{
+  margin-left:70px ;
+}
+.footer-input{
+  width: 75%;
+  height: 30px;
+}
+
+.container {
+  position: relative;
+  border: 3px solid orange;
+  height: 40px;
+  overflow: hidden;
+  margin-bottom: 10px;
+  width: 300px;
+}
+
+.text_input {
+  height: 40px;
+  width: 80%;
+  padding: 0;
+  line-height: 30px;
+  font-size: 20px;
+  padding: 0;
```

Comment:
remove this, you already write it above.

Reply:
I have modified it